tests/console/yast2_settings.pm

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: YaST writes config to 70-yast.conf and detects conflicts
+# - Install yast2-network
+# - verify writing config to the correct file
+# - ensure conflict detected when custom confing with high higher priority exist
+# - validate that YaST applies changes to the system. bsc#1167234
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use y2lan_restart_common;
+
+my $yastconf              = '/etc/sysctl.d/70-yast.conf';
+my $customconf_in_etc     = '/etc/sysctl.d/90-custom.conf';
+my $customconf_in_usr_lib = '/usr/lib/sysctl.d/90-custom.conf';
+
+sub verify_write_config {
+    open_yast2_routing();
+    send_key 'spc';
+    assert_screen "ipv4_forwarding_enabled";
+    send_key 'alt-n';
+    wait_serial("yast2-routing-status-0", 180) || die "'yast2-routing' didn't finish";
+    assert_script_run("sysctl net.ipv4.ip_forward | grep 1");
+}
+
+sub ensure_conflict_detected {
+    _create_conflicts();
+    open_yast2_routing();
+    send_key 'spc';
+    assert_screen "ipv4_forwarding_disabled";
+    send_key 'alt-n', wait_screen_change => 1;
+    assert_screen "sysctl_ip_forward_conflict";
```

Comment:
We are not checking the files conflicting with the needle, we could have a needle with a name indicating exact conflict for example. We could use the constant mentioned above to do it. As you verify later that the system didn't change this would be optional, of course...just an enhancement.

Reply:
lets discuss this. Because there is another approach on this that might be better. the solution invokes the output in the logs.

Reply:
As you see I have added two addtional scripts that validates the output from the logs. So in this approach we validate that the conflict occurs in the first file but not to the other.